### PR TITLE
Fix/dispatches

### PR DESCRIPTION
**Pull request by iparask**
Makes sure a single workflow is triggered by each resource and that the badges are on the main README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +31,10 @@ supported backends, please refer to the documentation.
 The `devel` branch (and any branch other than master) may not correspond to the
 published documentation and, specifically, may have dependencies which need to
 be resolved manually.
+
+## Integration Tests status
+These badges show the state of the current integration tests on different HPCs RADICAL Pilot supports
+
+[![ORNL Summit Integration Tests](https://github.com/iparask/git_actions/actions/workflows/summit.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/summit.yml)
+[![PSC Bridges2 Integration Tests](https://github.com/iparask/git_actions/actions/workflows/bridges.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/bridges.yml)
+[![SDSC Comet Integration Tests](https://github.com/iparask/git_actions/actions/workflows/comet.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/comet.yml)
```
Comment (mtitov) [inline README.md:40]: wrong paths for badges?

**Comment by iparask** [comment]
I updated the badges, the event triggers and added more information in the integration test readme file.

I changed the event names and have them unique per resource. I found that all three workflows were triggered by the same event name.

**Review by andre-merzky — APPROVED** [review]
LGTM, thanks!

File: README.md [hunk 2]
```diff
@@ -31,3 +31,10 @@ supported backends, please refer to the documentation.
 The `devel` branch (and any branch other than master) may not correspond to the
 published documentation and, specifically, may have dependencies which need to
 be resolved manually.
+
+## Integration Tests status
+These badges show the state of the current integration tests on different HPCs RADICAL Pilot supports
+
+[![ORNL Summit Integration Tests](https://github.com/iparask/radical-cybertools/radical.pilot/actions/workflows/summit.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/summit.yml)
+[![PSC Bridges2 Integration Tests](https://github.com/iparask/radical-cybertools/radical.pilot/actions/workflows/bridges.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/bridges.yml)
+[![SDSC Comet Integration Tests](https://github.com/iparask/radical-cybertools/radical.pilot/actions/workflows/comet.yml/badge.svg)](https://github.com/radical-cybertools/radical.pilot/actions/workflows/comet.yml)
```
Comment (mtitov) [inline README.md:40]: @iparask sorry, your name is still in urls for badges (`iparask`)

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**